flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/DecimalCastTest.scala

```diff
@@ -0,0 +1,203 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.expressions
+
+import org.apache.flink.api.java.typeutils.RowTypeInfo
+import org.apache.flink.table.planner.expressions.utils.ExpressionTestBase
+import org.apache.flink.types.Row
+
+import org.junit.Test
+
+import scala.util.Random
+
+class DecimalCastTest extends ExpressionTestBase {
```

Comment:
add types that don't support casting between decimal?

Reply:
Forgot this